src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -181,6 +181,15 @@ private bool UsePlaceholderChannel(IRpcWorkerChannel channel)
                     return false;
                 }
 
+                // We support specialization of dotnet-isolated only on 64bit host process.
+                var is64Bit = _environment.Is64BitProcess;
+                _logger.LogDebug("Is64BitProcess: {is64Bit}", is64Bit.ToString());
```

Comment:
let's add a more detailed log here. we want to inform customers in Risk Alerts if they can't use placeholders because their process is 64bit based on this log. this log is a bit too generic and might not be specific enough for this purpose.